moving-to-uk.md

```diff
@@ -0,0 +1,29 @@
+# Moving to UK from overseas
+
+Tips from our collective experience of moving to the UK.
+
+If you're moving to the USA, [look here](/moving-to-usa.md).
+
+* VISAs
```

Comment:
`VISA` → `Visa`; I think `VISA` is the card company and `Visa` isn't normally capitalised fully?